discord/ui/select.py

```diff
@@ -306,6 +357,132 @@ def options(self, value: list[SelectOption]):
 
         self._underlying.options = value
 
+    @property
+    def default_values(self) -> list[SelectDefaultValue]:
+        """A list of the select's default values. This is only applicable if
+        the select type is not :attr:`discord.ComponentType.string_select`.
+
+        .. versionadded:: 2.7
+        """
+        return self._underlying.default_values
+
+    @default_values.setter
+    def default_values(
+        self, values: list[SelectDefaultValue | Snowflake] | None
+    ) -> None:
+        default_values = self._handle_default_values(values, self.type)
+        self._underlying.default_values = default_values
+
+    def add_default_value(
+        self,
+        *,
+        id: int,
+        type: SelectDefaultValueType = MISSING,
+    ) -> Self:
+        """Adds a default value to the select menu.
+
+        To append a pre-existing :class:`discord.SelectDefaultValue` use the
+        :meth:`append_default_value` method instead.
+
+        .. versionadded:: 2.7
+
+        Parameters
+        ----------
+        id: :class:`int`
+            The ID of the entity to add as a default.
+        type: :class:`discord.SelectDefaultValueType`
+            The default value type of the ID. This is only required if :attr:`.type` is of
+            type :attr:`discord.ComponentType.mentionable_select`.
+
+        Raises
+        ------
+        TypeError
+            The select type is a mentionable_select and type was not provided, or the select
+            type is string_select.
+        ValueError
+            The number of default select values exceeds 25.
+        """
+        if type is MISSING and self.type is ComponentType.mentionable_select:
```

Comment:
A check similar to this should maybe be done earlier in `ui.Select` itself because there it directly references the `type` argument but someone that is just passing ids might not even realize that `SelectDefaultValue` exists